pkg/ebpf/c/maps.h

```diff
@@ -25,6 +25,13 @@
         __type(value, _value_type);                                                                \
     } _name SEC(".maps");
 
+#define BPF_MAP_QUEUE(_name, _type, _value_type, _max_entries)                                     \
+    struct {                                                                                       \
+        __uint(type, _type);                                                                       \
+        __uint(max_entries, _max_entries);                                                         \
+        __type(value, _value_type);                                                                \
+    } _name SEC(".maps");
+
```

Comment:
No need for this macro - `BPF_MAP` macro should suffice

Reply:
Tried that at first - supplying a key for BPF_MAP_TYPE_QUEUE maps seems to throw an error:
{"level":"warn","ts":1677430230.0956712,"msg":"libbpf: map 'found_hidden_modules_queue': failed to create: Invalid argument(-22)"}

Reply:
u32, u64...
![image](https://user-images.githubusercontent.com/39296766/226195330-4fd620b5-52b5-4d0b-8a3d-4c8f81403f15.png)

Reply:
key size must be zero - https://docs.kernel.org/bpf/map_queue_stack.html

Reply:
So the BPF_MAP_QUEUE macro shouldn't really be used by the user, rather the BPF_QUEUE which does not need the type.
I've defined the BPF_MAP_QUEUE just as a mean to support other use cases, for example a stack. Perhaps I'll rename it to something more relevant though, and add support for stack.